llm/llama.go

```diff
@@ -631,34 +632,37 @@ func (llm *llama) Predict(ctx context.Context, predict PredictOpts, fn func(Pred
 				continue
 			}
 
-			if evt, ok := bytes.CutPrefix(line, []byte("data: ")); ok {
```

Comment:
This diff is hard to read. I changed the logic to return the error when the response stream can't be parsed.